### PR TITLE
SM: check the username to allow for updates and removal of items

### PR DESCRIPTION
**Pull request by smathis122**
View Items Tab only shows items that were added by a specific user. Only that user will be allowed to update the item or remove the item

### CODE REVIEW

File: code/Backend/flask-backend/app.py [hunk 1]
```diff
@@ -30,14 +29,9 @@
...
 
-# Database table for user
-class User(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
-    email = db.Column(db.String(20), nullable=False, unique=True)
-    password = db.Column(db.String(200), nullable=False)
-
-
 CORS(app) # Allow all origins for development; restrict in production
+secret_key = secrets.token_hex(16)  # Generate a 32-character (16 bytes) random hexadecimal string
```
Comment (casejade37) [inline code/Backend/flask-backend/app.py:33]: Nice inline comment

File: code/Backend/flask-backend/app.py [hunk 2]
```diff
@@ -294,12 +270,6 @@ def login():
     except Exception as e:
         return jsonify({"error": str(e)}), 500
 
-@app.route('/api/dashboard', methods=['GET', 'POST'])
```
Comment (casejade37) [inline code/Backend/flask-backend/app.py:297]: Do we need to create a new dashboard page?

**Review by casejade37 — APPROVED** [review]
I feel the refactoring is already underway e.g. removing obsolete comments - remember to take screenshots when you bump into refactoring parts that meet Lab3 requirements.

**Comment by smathis122** [comment]
> I feel the refactoring is already underway e.g. removing obsolete comments - remember to take screenshots when you bump into refactoring parts that meet Lab3 requirements.

I agree had that in mind when deleting those comments! Thanks for the heads up though I appreciate it

File: code/Backend/flask-backend/app.py (hunk 2 above)
Comment (smathis122) [reply code/Backend/flask-backend/app.py:297]: It wasn't being used -- we have a home.js that does the home page stuff

**Review by lazpz61BU — APPROVED** [review]
In had the same Axios issue was I resolved by uploading, only thing is when I pulled into this branch the file was still navbar.js and not Nabar.js no biggie, I had just thought that it was updated from the communication on discord between you and Ahnaf. Was able to register, and add and item on the UI, one thing I will mention is that when I added and item and then deleted, when I went to search I noticed that a shell of where the item had been populated, so doing this test allowed me to think about this scenario and now I have something to consider when doing the search for items that need to have information. I can demo what Im seeing on our team call.

**Pull request merged**